advanced/am-pattern-2/values.yaml

```diff
@@ -202,138 +159,154 @@ wso2:
-#        config:
-#          deployment.yaml: |-
-#            # deployment configurations for the Worker profile of WSO2 API Manager Analytics v3.2.0 (<WSO2AM_ANALYTICS>/conf/worker/deployment.yaml)
-
+        # These are the minimum resource recommendations for running WSO2 Micro Integrator
         resources:
-          # These are the resource recommendations for running WSO2 Stream Processor based server profiles
-          # as per official documentation (https://docs.wso2.com/display/SP440/Installation+Prerequisites).
           requests:
-            memory: "4Gi"
-            cpu: "2000m"
-          # Please see the official documentation on WSO2 Stream Processor based Performance Analysis and Resource recommendations
-          # (https://docs.wso2.com/display/SP440/Performance+Analysis+Results) and tune the limits according to your needs
-          # where necessary.
+            # The minimum amount of memory that should be allocated for a Pod
+            memory: "512Mi"
+            # The minimum amount of CPU that should be allocated for a Pod
+            cpu: "500m"
           limits:
-            memory: "4Gi"
-            cpu: "2000m"
-          # JVM settings
-          # These are the resource allocation configurations associated with the JVM
-          # Refer to the official documentation for advanced details (https://docs.wso2.com/display/SP440/Installation+Prerequisites)
-          jvm:
-            # Resource allocation for the Java Heap
-            heap:
-              memory:
-                # Initial and minimum Heap size
-                xms: "2048m"
-                # Maximum Heap size
-                xmx: "2048m"
-
-kubernetes:
-  # Name of Kubernetes service account
-  serviceAccount: "wso2am-pattern-2-svc-account"
-
-# Override sub chart parameters
-mysql-am:
-  mysql:
-    persistence:
-      storageClass: *storage_class
+            # The maximum amount of memory that should be allocated for a Pod
+            memory: "1Gi"
+            # The maximum amount of CPU that should be allocated for a Pod
+            cpu: "1000m"
+        # Environment variables for the Micro integrator deployment.
+        envs:
+        #  ENV_NAME: ENV_VALUE
+        # Configure synapse testing.
+        synapseTest:
+          enabled: false
+
+        # Configure Ingresses
+        ingress:
+          # Configure management ingress
+          management:
+            # Hostname for the Micro Integrator management endpoint.
+            hostname: "tenant2.management.mi.wso2.com"
+            # Annotations for the Micro Integrator management Ingress.
+            annotations:
+              kubernetes.io/ingress.class: "nginx"
+              nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+
+  kubernetes:
+    # Name of Kubernetes service account
+    serviceAccount: *service_account
+
+  # Override sub chart parameters
+  mysql-am:
+    mysql:
+      persistence:
```

Comment:
Do we need this part when using pattern-1 as the subchart?

Reply:
yes, if they want to configure the **storageClass** of pattern-1. they can do it from here.